code/game/gamemodes/miniantags/borer/borer.dm

```diff
@@ -42,20 +42,18 @@
 	to_chat(src, "<span class='danger'>You begin doggedly resisting the parasite's control (this will take approximately sixty seconds).</span>")
 	to_chat(B.host, "<span class='danger'>You feel the captive mind of [src] begin to resist your control.</span>")
 
-	spawn(rand(350,450) + B.host.brainloss)
+	var/delay = (rand(350,450) + B.host.brainloss)
```

Comment:
Are you sure? I made this shorter on TG because it's more fast paced and you're less likely to get surgery done